lib/screens_web/views/v2/audio/reconstructed_alert_fullscreen_view.ex

```diff
@@ -0,0 +1,206 @@
+defmodule ScreensWeb.V2.Audio.ReconstructedAlertFullscreenView do
+  use ScreensWeb, :view
+
+  alias Screens.Alerts.Alert
+  alias Screens.Util
+
+  def render("_widget.ssml", alert) do
+    ~E|<p><%= render_banner(alert) %><%= render_alert(alert) %></p>|
+  end
+
+  def render_banner(%{
```

Comment:
This case is for a fullscreen alert, right? Since the `:unaffected_routes` key gets set only in that case? If so, a comment would help here. The significance of that field is not clear.